exporter/tanzuobservabilityexporter/metrics_exporter.go

```diff
@@ -0,0 +1,92 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tanzuobservabilityexporter
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+	"strconv"
+
+	"github.com/wavefronthq/wavefront-sdk-go/senders"
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/config"
+	"go.opentelemetry.io/collector/model/pdata"
+)
+
+type metricsExporter struct {
+	consumer *metricsConsumer
+}
+
+func createMetricsConsumer(
+	hostName string, port int, settings component.TelemetrySettings) (*metricsConsumer, error) {
+	s, err := senders.NewProxySender(&senders.ProxyConfiguration{
+		Host:                 hostName,
+		MetricsPort:          port,
+		FlushIntervalSeconds: 1,
+	})
+	if err != nil {
+		return nil, fmt.Errorf("failed to create proxy sender: %v", err)
+	}
+	cumulative := newCumulativeHistogramDataPointConsumer(s)
+	delta := newDeltaHistogramDataPointConsumer(s)
+	return newMetricsConsumer(
+		[]typedMetricConsumer{
+			newGaugeConsumer(s, settings),
+			newSumConsumer(s, settings),
+			newHistogramConsumer(cumulative, delta, s, regularHistogram, settings),
+			newHistogramConsumer(cumulative, delta, s, exponentialHistogram, settings),
+			newSummaryConsumer(s, settings),
+		},
+		s,
+		true), nil
+}
+
+type metricsConsumerCreater func(hostName string, port int, settings component.TelemetrySettings) (
```

Comment:
should this be `metricsConsumerCreator`?

Reply:
Done. Thanks for catching this.